microsoft/PowerToys
# [Screen Reader - Utility - PowerToys Run] Mention context menu items

## Summary of the Pull Request

Mention how many menu items the element has, so users know about them and may investigate them when needed. 


## PR Checklist

- [x] **Closes:** #20270 


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed

* Check with something with context menu items, e.g. Word - Narrator says "Word, application, 4 context menu items available"
* Check something without context menu items (hibernate), no context menu items mentioned
